docker/build_jakartaeetck.sh

```diff
@@ -94,7 +94,7 @@ wget --progress=bar:force --no-cache $GF_BUNDLE_URL -O latest-glassfish.zip
 unzip -o latest-glassfish.zip
 ls -l $GF_HOME/glassfish5/glassfish/
 
-#temporary fix to get build passing until we have glassfish with new api jars
+# temporary fix to get build passing until we have glassfish with new api jars
```

Comment:
Can you update the copyright year in the sh files, too?

Reply:
Done.